minigalaxy/ui/gametile.py

```diff
@@ -169,6 +170,13 @@ def __set_image(self):
             return True
         return False
 
+    def get_keep_executable_path(self):
+        if os.path.exists(self.keep_path):
+            for fil in os.scandir(self.keep_path):
```

Comment:
This causes an exception for me:
```
Traceback (most recent call last):
  File "/home/wouter/Sources/minigalaxy/minigalaxy/ui/library.py", line 108, in __create_gametiles
    self.__add_gametile(game)
  File "/home/wouter/Sources/minigalaxy/minigalaxy/ui/library.py", line 111, in __add_gametile
    self.flowbox.add(GameTile(self, game, self.api))
  File "/home/wouter/Sources/minigalaxy/minigalaxy/ui/gametile.py", line 64, in __init__
    self.reload_state()
  File "/home/wouter/Sources/minigalaxy/minigalaxy/ui/gametile.py", line 265, in reload_state
    elif self.get_keep_executable_path():
  File "/home/wouter/Sources/minigalaxy/minigalaxy/ui/gametile.py", line 175, in get_keep_executable_path
    for fil in os.scandir(self.keep_path):
NotADirectoryError: [Errno 20] Not a directory: '/home/wouter/Games/installer/METAL SLUG'
```

Reply:
To add to this, the rest of the gametiles are not being loaded because of this exception as well, so this is critical.

Reply:
Ah, I see, that's an oversight by me, because I didn't have any installers saved before I made the change. Old installers have the same name as the directory where the new installers should be located, so that problem should only arise for old installer files.
I will add a quick directory check to fix that.